schema.graphql

```diff
@@ -475,3 +475,14 @@ type SingleSidedStakingCampaignPosition @entity {
   stakedAmount: BigDecimal!
   singleSidedStakingCampaign: SingleSidedStakingCampaign!
 }
+
+type BlockPairTokenPrice @entity {
+  id: ID!
+  blockNumber: BigInt!
+  blockTimestamp: BigInt!
+  pair: Pair!
+  token0Price: BigDecimal!
+  token0Address: Bytes!
+  token1Price: BigDecimal!
+  token1Address: Bytes!
+}
```

Comment:
Missing newline at end of file. :)

Reply:
Fixed in [fe8aeca](https://github.com/dxgraphs/swapr-subgraph/pull/23/commits/fe8aecae915fe290462a0f652681a4a6f51b3575)